coinbase/wallet/client.py

```diff
@@ -33,10 +35,17 @@
 from coinbase.wallet.util import check_uri_security
 from coinbase.wallet.util import encode_params
 
+from Crypto.Hash import SHA256
+from Crypto.PublicKey import RSA
+from Crypto.Signature import PKCS1_v1_5
 
 COINBASE_CRT_PATH = os.path.join(
     os.path.dirname(os.path.realpath(__file__)), 'ca-coinbase.crt')
 
+COINBASE_CALLBACK_PUBLIC_KEY_PATH = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), 'coinbase-callback.pub')
+
+PY3 = sys.version_info > (3,)
```

Comment:
I would make this the same way as in compat.py: https://github.com/coinbase/coinbase-python/blob/master/coinbase/wallet/compat.py